### PR TITLE
Cookiecutting - mod files 1

### PR DESCRIPTION
**Pull request by 8bitsam**
Split the other PR into two smaller ones.

### CODE REVIEW

**Comment by sbillinge** [comment]
@8bitsam I created the new repo on diffpy for `diffpy.fourigui` using cookiecutter.

I am not sure the best way to merge things from here into there.  Maybe using a similar workflow, but moving the merged files over to the new repo instead of leaving them in diffpy.fourigui that you made locally in the current owrkflow?  What do you think?

The goals would be to try and make the workflow as easy as possible, but also to allow good review.  Maybe I could merge things here, and after they are merged you could move them over (though I used the latest version of cookiecutter with new things merged, so I would rather use as a starting point the files over there).

**Comment by 8bitsam** [comment]
Luckily this package seems easy for this. There's only one main class, `src/diffpy/fourigui/fourigui.py` at, which can be placed in the `diffpy.fourigui` repo at the same path. Then, there are the tests `unit_test.py` and `integration_test.py` that need to be merged, which strangely occur at two places in this repo, but we want to merge them into `src/diffpy/fourigui/tests/`.

All of the changes to the code files that we would have to copy over from black/flake8 fixes are done, so there's no need to merge anything here right now.

The last thing to merge is the `further_processing_code/` directory, which doesn't contain anything that's actually called by functions in the package, but still seems to have some usefulness? Maybe this could go in `doc/` at some location or just removed entirely since the code in there seems to reference files on someone's computer, which don't exist anywhere in this repo.

In short the main two things are `fourigui.py` and the two test files, which can be merged each in separate PRs probably to a new branch on `diffpy.fourigui`.
